Cabal/Distribution/Utils/Json.hs

```diff
@@ -0,0 +1,54 @@
+{-# LANGUAGE OverloadedStrings #-}
+-- | Extremely simple JSON helper. Don't do anything too fancy with this!
+
+module Distribution.Utils.Json
+    ( Json(..)
+    , (.=)
+    , renderJson
+    ) where
+
```

Comment:
Use `Distribution.Compat.Prelude` (look in other modules), that might make CI work (on older GHCs).